aiohttp/web_urldispatcher.py

```diff
@@ -428,6 +438,9 @@ def add_prefix(self, prefix):
         assert len(prefix) > 1
         self._prefix = prefix + self._prefix
 
+    def raw_match(self, prefix):
+        return self._prefix == prefix
```

Comment:
Reusing `StaticResource` and subapp resource is dangerous.
Let's always return False.